datafusion/core/tests/parquet/external_access_plan.rs

```diff
@@ -313,7 +313,7 @@ impl TestFull {
         } = self;
 
         let TestData {
-            temp_file: _,
+            _temp_file: _,
```

Comment:
I think this can be even dropped 🤔

Reply:
Thank you for your review. Just to clarify, do you mean this?
```rust
let TestData {
    schema,
    file_name,
    file_size,
    ..
} = get_test_data();
```

Reply:
I think the idea is that we need to keep the `_temp_file` around through the end of the function so that the temporary directory isn't deleted until the end of the function (when the `drop` is called)

Reply:
Oh, thats the thing... so we keeping it in scope to avoid the folder to be killed prematurely. LGTM then, adding some comments

Reply:
I updated the comments, but I'm a bit confused yet :cry: 
If we need to keep the `_temp_file` around through the end of the function `run()`, don't we need to actually bind `_temp_file` instead of  `_temp_file : _` ? Or is it fine to drop `_temp_file` right after `get_test_data()`?

Reply:
I tried it out as it wasn't obvious to me from the code -- it seems like `get_test_data()` actually returns a *reference* to `&'static TestData` -- so it doesn't actually matter if the `_temp_file` field is bound or not locally - `Drop` isn't done until the process quits

Reply:
Ahh I see. Thank you.